### PR TITLE
PIPE2D-1063: Tune CR parameters for simulator data

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

File: include/pfs/drp/stella/traces.h [hunk 1]
```diff
@@ -106,6 +107,24 @@ void centroidPeak(
...
+/// This is intended to remove the traces, to allow the cosmic ray finder to
+/// operate on an image (more) devoid of continuum sources which can cause
+/// false positive CRs.
+///
+/// @param image : Image to which to apply the median filter
+/// @param mask : Pixels that should be ignored (true means a bad value)
+/// @param halfHeight : Half-height of the median filter (pixels)
+/// @returns Median filter image
+template <typename T>
+ndarray::Array<T, 2, 1> medianFilterColumns(
```
Comment (hassanxp) [inline include/pfs/drp/stella/traces.h:121]: Is this code actually used in your CR detection algorithm? I couldn't see the call.

**Review by hassanxp — APPROVED** [review]
Changes look fine. Minor comments added.

File: python/pfs/drp/stella/repair.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            high = min(exposure.getHeight(), yy + self.config.halfHeight + 1)
+            array = exposure.image.array[low:high, :]
+            background[yy, :] = np.median(array, axis=0)
+
+        # Find CRs in background-subtracted image
+        exposure.image.array -= background
+        try:
+            cosmicrays = findCosmicRays(exposure.maskedImage, psf, 0.0, config, keepCRs)
+        finally:
+            exposure.image.array += background
```
[the remark this answers is not in the document]
Comment (PaulPrice) [reply python/pfs/drp/stella/repair.py]: That would be best if the cosmic ray finder didn't modify the pixels.

File: include/pfs/drp/stella/traces.h (hunk 1 above)
Comment (PaulPrice) [reply include/pfs/drp/stella/traces.h:121]: Right [here](https://github.com/Subaru-PFS/drp_stella/blob/tickets/PIPE2D-1063/python/pfs/drp/stella/repair.py#L95).

File: python/pfs/drp/stella/repair.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (PaulPrice) [reply python/pfs/drp/stella/repair.py]: Sorry, forgot to mention that we may want the CR finder's modification of the pixels. If configured appropriately, it interpolates over the CR pixels, which is something we would want to preserve.

**Pull request merged**